core/tests/protractor/stateEditor.js

```diff
@@ -134,6 +134,30 @@ describe('State editor', function() {
     users.logout();
   });
 
+  it('should skip the customization modal for interactions having no ' +
```

Comment:
Yes, and it would be good to make sure it stays that way :P Cancelling a change should also work correctly.